### ISSUE TITLE
Unexpected token is happened in v1.5.9 

### ISSUE THREAD

**Issue by shiganaiman** [cmp-cc/vue-cookies#24]
Hi.
We can not build  the latest version(v1.5.9),
because "Unexpected token  error" is happened.
Please fix this bug.

```
Module parse failed: Unexpected token (35:109)
You may need an appropriate loader to handle this file type.
|             var value = decodeURIComponent(document.cookie.replace(new RegExp("(?:(?:^|.*;)\\s*" + encodeURIComponent(key).replace(/[\-\.\+\*]/g, "\\$&") + "\\s*\\=\\s*([^;]*).*$)|^.*$"), "$1")) || null
|
>             if(value && value.substring(0,1) === "{" && value.substring(value.length-1,value.length === "}") {
|                 try {
|                     value = JSON.parse(value)
```
https://github.com/cmp-cc/vue-cookies/blob/b74ad51f5d2d61c5eb9ca518518662d30a6a3007/vue-cookies.js#L35

**Comment by maverickpravin** [comment]
Same error with me  :(

**Comment by cmp-cc** [comment]
i'm sorry 

fix v1.5.10
